reference/solver/ir_kernels.cpp

```diff
@@ -36,6 +36,11 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace gko {
 namespace kernels {
 namespace reference {
+/**
+ * @brief The IR solver.
+ *
```

Comment:
No `@ref` like the other solvers?

Reply:
The `@ref` is actually not needed when `@ingroup` in there. I will remove it in all others. Thanks for pointing this out.